src/cockpit/samples.py

```diff
@@ -398,7 +398,15 @@ def sample(self, samples: Samples) -> None:
                 if line[0] != '/':
                     continue
 
-                path = line.split()[1]
+                split_line = line.split()
+                # Skip read-only loop mounts
+                if '/loop' in split_line[0] and 'ro' in split_line[3].split(','):
```

Comment:
Wouldn't it be nicer to properly decompose this

```python
mounts = open('/proc/mounts').read()
fs_spec, fs_file, fs_vfstype, fs_mntopts, *_rest  = mounts.splitlines()[0].split()
```

Not sure if one can remove `*_rest`, field definitions comes from https://man.archlinux.org/man/fstab.5